jakttest/jakttest.jakt

```diff
@@ -633,13 +647,15 @@ struct TestScheduler {
         total_test_count: usize
         build_dir: String
         cpp_compiler_path: String? = None
+        opt_target_triple: String? = None
```

Comment:
:thinking: nothing is passing a value to this though?

Reply:
looks like I reverted changes when trying to move the changes to a separate branch. Lemme fix this

Reply:
I fixed it and it turns out to work inside the nix shell without any target triple overloading. I'll close this since it's not needed after all.